pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -46,7 +46,7 @@ one = 'Format in which to display the Kafka instances (choose from: "json", "yml
 
 [kafka.common.error.couldNotFetchKafkas]
 description = 'Error message when list of Kafka instances could not be fetched'
-one = 'unable to list Kafka instances: {{.Error}}'
+one = 'unable to list Kafka instances'
```

Comment:
Is this used anywhere anymore?

Reply:
It is not used, but it was there, so not sure whether I should delete it or not.

Reply:
Kill it with fire. Then we merge PR.

Reply:
Done.